Mathlib/Analysis/Analytic/Meromorphic.lean

```diff
@@ -310,6 +327,82 @@ lemma id {U : Set 𝕜} : MeromorphicOn id U := fun x _ ↦ .id x
 lemma const (e : E) {U : Set 𝕜} : MeromorphicOn (fun _ ↦ e) U :=
   fun x _ ↦ .const e x
 
+/-- The set where a meromorphic function has infinite order is clopen in its domain of meromorphy.
+-/
+theorem isClopen_setOf_order_eq_top {U : Set 𝕜} [CompleteSpace E] (hf : MeromorphicOn f U) :
```

Comment:
ditto, completeness is probably not necessary.

Reply:
Implemented to the best of my abilities. I won't be winning points for beauty & elegance, though. You will definitively want to look at this.